main/guides/orchestration/getting-started/contract-walkthroughs.md

```diff
@@ -31,9 +43,9 @@ The Unbond Contract focuses on the process of unbonding staked assets and perfor
 
 [See Contract Overview](/guides/orchestration/getting-started/contract-walkthrough/cross-chain-unbond)
 
-## Dapp Orchestration Basics
+## DApp Orchestration Basics
 
-The Agoric Dapp Orchestration Basics walkthrough (currently under development) will provide an introduction to the core concepts and basic implementation of orchestrtion within an Agoric dApp. This guide aims to:
+DApp Orchestration Basics walkthrough will provide an introduction to the implementation level details of orchestration within an Agoric dApp. This guide aims to:
```

Comment:
I don't think it currently shows what we say here that it shows. But that's a discussion/modification for a different PR.

Reply:
Removing it since we are moving it from example contracts anyway in another PR.